plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/SelectorSpec.java

```diff
@@ -84,6 +87,11 @@ public Optional<Pattern> getSourceRegex()
         return sourceRegex;
     }
 
+    public Optional<Pattern> getQueryText()
```

Comment:
The sql itself has "*" for representing the all columns, while I am curious what's the behavior for query contains "*"